mfs/ops.go

```diff
@@ -100,9 +100,9 @@ func PutNode(r *Root, path string, nd *dag.Node) error {
 
 // Mkdir creates a directory at 'path' under the directory 'd', creating
 // intermediary directories as needed if 'mkparents' is set to true
-func Mkdir(r *Root, pth string, mkparents bool, flush bool) error {
+func Mkdir(r *Root, pth string, mkparents bool, flush bool) (*Directory, error) {
 	if pth == "" {
-		return nil
+		return nil, nil
```

Comment:
returning nil and nil feels weird to me... code might assume that no error means you get back a valid directory.